clang/include/clang/Basic/Features.def

```diff
@@ -308,6 +308,10 @@ EXTENSION(datasizeof, LangOpts.CPlusPlus)
 
 FEATURE(cxx_abi_relative_vtable, LangOpts.CPlusPlus && LangOpts.RelativeCXXABIVTables)
 
+//Fixed enum feature and extension, to be relocated in this file
+FEATURE(c_fixed_enum, true)    
+EXTENSION(c_fixed_enum, true)  
```

Comment:
Haven’t looked at the features code in a bit, but does this need to be both a FEATURE and an EXTENSION?

CC @AaronBallman

Reply:
Yeah, it should be both, but I'd move the code around a bit, similar to:

https://github.com/llvm/llvm-project/blob/4ab298b5fbc8f48387062b2dd99ea07127c02e6b/clang/include/clang/Basic/Features.def#L160 (we can add a new section for C23 features)

https://github.com/llvm/llvm-project/blob/4ab298b5fbc8f48387062b2dd99ea07127c02e6b/clang/include/clang/Basic/Features.def#L263 (we can put under the C23 section)